statement.go

```diff
@@ -110,3 +116,45 @@ func (stmt *mysqlStmt) Query(args []driver.Value) (driver.Rows, error) {
 
 	return rows, err
 }
+
+type converter struct{}
+
+func (converter) ConvertValue(v interface{}) (driver.Value, error) {
+	if driver.IsValue(v) {
+		return v, nil
+	}
+
+	if svi, ok := v.(driver.Valuer); ok {
```

Comment:
The database/sql package already checks this case: https://github.com/golang/go/blob/master/src/database/sql/convert.go#L48